vNext/channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -307,7 +310,8 @@ export class Sender implements IChannelControlsAI {
      * Immediately send buffered data
      * @param async {boolean} - Indicates if the events should be sent asynchronously
      */
-    public triggerSend(async = true) {
+    public triggerSend(async?: boolean) {
+        if (!async) async = true;
```

Comment:
This will result in async always being true. Use [Typescript default parameters instead](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters)